pkg/jobs/job_scheduler.go

```diff
@@ -314,10 +276,10 @@ func (s *jobScheduler) executeSchedules(
 				return contextutil.RunWithTimeout(
 					ctx, fmt.Sprintf("process-schedule-%d", schedule.ScheduleID()), timeout,
 					func(ctx context.Context) error {
-						return s.processSchedule(ctx, schedule, numRunning, stats, txn)
+						return s.processSchedule(ctx, schedule, numRunning, &stats, txn)
```

Comment:
Why did the `stats` argument change to a pointer? Looks like it might previously have been incorrect?

Reply:
That's fine; we no longer have newLoopStats -- so, loopStats are now stack allocated in this function, but are passed
to process scheduler as a pointer since process scheduler mutates loop stats.